src/text-editor-component.js

```diff
@@ -804,7 +804,15 @@ class TextEditorComponent {
           key: overlayProps.element,
           overlayComponents: this.overlayComponents,
           measuredDimensions: this.overlayDimensionsByElement.get(overlayProps.element),
-          didResize: () => { this.updateSync() }
+          didResize: (overlayComponent) => {
```

Comment:
Good idea passing in the component reference in this callback. This is such a twisty data relationship but I think this is a reasonable way to navigate it.